microsoft-graph.d.ts

```diff
@@ -1,10 +1,11 @@
-// Project: https://github.com/microsoftgraph/msgraph-beta-typescript-typings
+// Project: https://github.com/microsoftgraph/msgraph-typescript-typings
```

Comment:
I think I asked this before and perhaps I didn't see the answer. Why is this change here?

Reply:
@nikithauc, I think we need to fix this in the generator.
Our original comments are here: https://github.com/microsoftgraph/msgraph-beta-typescript-typings/pull/38#discussion_r701589147. I think the bug description there doesn't address the repo name.